llama_index/indices/composability/graph.py

```diff
@@ -16,10 +17,12 @@ def __init__(
         self,
         all_indices: Dict[str, BaseGPTIndex],
         root_id: str,
+        storage_context: Optional[StorageContext] = None,
```

Comment:
Just double checking, but I'm assuming this simplifies how persist works, so that it's similar to regular indexes?

`graph.storage_context.persist(...)`

Reply:
Yeah it's done to make it similar to regular indexes.